packages/trader/src/Stores/Modules/SmartChart/smart-chart-store.js

```diff
@@ -334,6 +336,8 @@ export default class SmartChartStore extends BaseStore {
     // ---------- WS ----------
     wsSubscribe = (request_object, callback) => {
         if (request_object.subscribe !== 1) return;
+        if (this.is_contract_mode && !this.is_tick_granularity && !request_object.granularity) return;
+        console.log(request_object);
```

Comment:
do we need to log?, maybe can add a comment

Reply:
It's still [WIP] you're not supposed to review this yet. 😅